api/src/api/pitch.ts

```diff
@@ -407,6 +417,7 @@ router.put(
       {
         $pull: {
           pendingContributors: { userId: userId },
+          submittedClaims: req.params.pitchId,
```

Comment:
Would you pull this from the Pitch model? Isn't submittedClaims stored in the User model?

Reply:
Nice catch.